packages/ui/package.json

```diff
@@ -67,6 +67,8 @@
     "react-router-dom": "^4.3.1"
   },
   "dependencies": {
+    "@syndesis/models": "*",
```

Comment:
I think we wanted to avoid binding the UI widgets to the model ideally.  I think this was added to support IExtensionDetailProps `extension` member but I kinda recall we want to let the UI widgets have their own interface/props that's decoupled from the backend model.